### ISSUE TITLE
What do you mean ebow isn't good?

### ISSUE THREAD

**Issue by ShadowDeath6** [yogstation13/yogstation-classic#835]
tl;dr shadow doesn't like combo cafe and wants to make ebow an actual viable weapon

As I'm writing this issue (I'm fiddling with damage) ebow will do 20 toxin damage on hit, stun for 1 tick instead of weaken for 5, and does 30 radiation damage. Since there's talk of revolver being lowered to like 11 TC, I was thinking ebow could actually become its own weapon instead of being a supplementary "shoot then stab with generic weapon with decent damage".

Currently, around 4 shots will cause them to start collapsing from radiation and toxins, and 5 will crit. I think this is fair given the decreased stun.

Also, ebows now recharge automatically. (as do kinetic accelerators)

**Comment by FluffySurvivor** [comment]
Imo, make the ebow have an automatic recharge system like oldebow. Cap it at say 3 bolts, then make the bolts shoot some delayed paralyzing chem.

Voila, no more ebow cheese and the ebow actually becomes a stealth-exclusive item (IMAGINE SHOOTING SOMEONE FROM THE DARKNESS OF MAINT THEN STALK HIM UNTIL HE FALLS DOWN OH BOY THE HYPE)

**Comment by darkwahn** [comment]
Ebow is already an auto-win in any 1v1 situation though. Even if it does take... a while.
